docs/guides/src/main/server/keycloak-truststore.adoc

```diff
@@ -2,7 +2,7 @@
 <#import "/templates/kc.adoc" as kc>
 
 <@tmpl.guide
-title="Configuring a Truststore"
+title="Configuring a Truststore for Outgoing TLS connections"
```

Comment:
I don't think this is necessary as it's explained in the summary, and would make the title very long/ugly

Reply:
The summary already explains the purpose of configuring a trust store. The proposed change is basically making it more explicit when reading the title.

The problem here is that by reading `Configuring a Truststore` is not clear at all the intent of the guide. That is why I'm changing the title.

Reply:
What about:

```suggestion
title="Configuring trusted certificates for outgoing requests"
```